src/InteractiveCodeSearch.jl

```diff
@@ -47,23 +47,34 @@ using Base
 using Base: IOError
 using InteractiveUtils: edit, gen_call_with_extracted_types, methodswith
 
-function _readandwrite(cmds)
-    processes = open(cmds, "r+")
-    return (processes.out, processes.in, processes)
-end
-
-
 abstract type SearchPolicy end
 struct Shallow <: SearchPolicy end
 struct Recursive <: SearchPolicy end
 
 
 mutable struct SearchConfig  # CONFIG
     open
-    interactive_matcher::Cmd
+    interactive_matcher::Function
```

Comment:
This breaks backward-compatibility.  Let's still support

    CONFIG.interactive_matcher = 

and just remove `::Cmd`.  The idea is to use run-time dispatch: https://github.com/tkf/InteractiveCodeSearch.jl/issues/6#issuecomment-574494692